spec/poros/food_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Food do
+  xit 'exists and has proper  movie attributes' do
+    data = {
+      "fdcId": 1100678,
```

Comment:
If gtinUpc and brandOwner are not always present, how can you build in a nil check to your PORO to cover the cases in which the hash looks like this one (without the attributes)